design-proposals/project-deploymnet.md

```diff
@@ -0,0 +1,174 @@
+# Design Proposal: Project Deployment with Kubernetes Operator
+
+**Author**: @kfirtoledo
+
+**Begin Design Discussion**: 2023-11-30
+
+**Status:** draft
+
+## Summary/Abstract
+
+This design proposal outlines the deployment process for the ClusterLink project on a Kubernetes (K8s) cluster.  
+The proposal specifically addresses deployment within the K8s environment.\
+It will rely on a dedicated K8s operator, to make the deployment more user-friendly, simplified and native to K8s users.
+
+## Background
+
+### Motivation and problem space
+
+This design proposal aims to simplify and enhance the deployment of the ClusterLink project.
+By leveraging K8s native mechanisms, such as operators and CRDs, the objective is to establish a user-friendly deployment process.\
```

Comment:
More trailing `\` at end of line. This seems to be repeated throughout the document and should be fixed..